BUILD.bazel

```diff
@@ -35,8 +35,5 @@ cc_library(
     srcs = [
         "stout/scheduler.cc",
     ],
-    deps = [
```

Comment:
This will break Linux and Mac OS builds now?

Reply:
It is cancelled.